src/game/texscroll/ccm_texscroll.inc.c

```diff
@@ -9,4 +9,5 @@ void scroll_textures_ccm() {
 	scroll_ccm_level_geo_tank_base();
 	scroll_ccm();
 	scroll_ccm_level_geo_rmine();
+	scroll_ccm();
```

Comment:
actually your level is sextuple scrolling now monkaS

Reply:
wouldn't this be triple scrolling

Reply:
anyway it fixed